.readthedocs.yml

```diff
@@ -0,0 +1,11 @@
+# Read the Docs configuration file
+version: 2
+
+sphinx:
+  builder: html
+  configuration: docs/conf.py
+
+python:
+  version: 3.8
```

Comment:
Any reason not to use 3.9?

Reply:
Nope! rtd was a little late to support python 3.9, but it looks like it's supported now: https://github.com/readthedocs/readthedocs.org/issues/7554